networkx/algorithms/similarity.py

```diff
@@ -1080,7 +1101,8 @@ def get_edit_paths(
             ]
         ).reshape(m, n)
         if roots:
-            initial_cost = 1 - node_match(G1.nodes[root_u], G2.nodes[root_v])
+            initial_cost = sum(
+                [1 - node_match(G1.nodes[root_u], G2.nodes[root_v]) for root_u, root_v in roots])
```

Comment:
Should this be (1 - sum(node_match for all pairs of root nodes))  or  sum( 1 - nodematch)??